### PR TITLE
Add build scripts

### PR DESCRIPTION
**Pull request by mschuettlerTNG**
**Description:**

This PR
* adds the build scripts for AI Playground
* updates some dependencies
* adds a theme for future use

- [x] I have tested the changes locally.
- [x] I have self-reviewed the code changes.
- [x] I have updated the documentation, if necessary.

### CODE REVIEW

File: WebUI/electron/main.ts [hunk 1]
```diff
@@ -537,6 +537,17 @@ function isProcessRunning(pid: number) {
...
+  try {
+    // replace `torchvision.transforms.functional_tensor` with `torchvision.transforms.functional` in `degradations.py`
+    const basicSrPathFileToBePatched = path.resolve(path.join(baseDir, "env/Lib/site-packages/basicsr/data/degradations.py"));
+    const fileContent = fs.readFileSync(basicSrPathFileToBePatched, 'utf8');
+    const patchedContent = fileContent.replace('torchvision.transforms.functional_tensor', 'torchvision.transforms.functional');
+    fs.writeFileSync(basicSrPathFileToBePatched, patchedContent, 'utf8');
+  } catch (error) {
+    logger.error(`Could not patch torchvision.transforms.functional_tensor in degradations.py: ${error}`);
+  }
+
```
Comment (Nuullll) [inline WebUI/electron/main.ts:550]: I've just fixed this issue in https://github.com/intel/AI-Playground/commit/9cb36b407247acecccfdb1939671ed0e52624a1f (probably in a more portable way).
You can safely revert this change.

Comment (mschuettlerTNG) [reply WebUI/electron/main.ts]: Nice! Thank you!

**Review by Nuullll — APPROVED** [review]
